README.md

```diff
@@ -3,9 +3,14 @@ This is a wedding website for Krishi and Luv's wedding. It will allow hosts to l
 
 ### Developer Setup
 To run this application, you need to first make sure you have the correct version of Node installed:
-```bash
+```
+bash
 brew install nvm && nvm install 12.13.1
 ```
+- Postgres
+- Go - https://golang.org/doc/install
+- Postman (Or online if you prefer)
```

Comment:
They technically don't need this to run the app, we can put this later in the docs in "testing"